MapboxVision/Services/Recording/RecordingQuota.swift

```diff
@@ -48,20 +31,40 @@ final class RecordingQuota {
         }
     }
 
-    private var currentQuota: Int64 {
+    private var cachedCurrentQuota: Byte {
```

Comment:
I suggest to rename `Byte` into `SizeInBytes`, for me it would be more appropriate because byte interfere with type `Byte` which is int8 in some languages

Reply:
Agreed to use `MemoryByte` typealias.